rust/otap-dataflow/Cargo.toml

```diff
@@ -171,6 +171,11 @@ reqwest = { version = "0.12.24", default-features = false }
 time = "0.3.44"
 wiremock = "0.6.5"
 
+# KQL processor
+data_engine_recordset_otlp_bridge = { path = "../experimental/query_engine/engine-recordset-otlp-bridge" }
+data_engine_recordset = { path = "../experimental/query_engine/engine-recordset" }
+data_engine_expressions = { path = "../experimental/query_engine/expressions" }
```

Comment:
There are re-exports so you shouldn't need to manually reference recordset + expressions to use the bridge. Probably OK to have them if you want, just pointing that out 😄

https://github.com/open-telemetry/otel-arrow/blob/6dbec074678d2265b953223691466e41ffa0ce4b/rust/experimental/query_engine/engine-recordset-otlp-bridge/src/lib.rs#L18-L21